templates/storypages/css/index.css

```
@@ -68,3 +68,7 @@ main {
 .slide-nav-select {
   min-width: 0;
 }
+
+.img-eye{
+  width: 20px;
```

this image is appearing very small on my browser - I would increase the size. Any easy way to do this would be `width: 200px;`
however, px are an *absolute* unit and it would take some back-and-forth resizing/checking to find the perfect px amount. If you want to snap the image to the width of the slide content, just put in a *relative* unit such as %
```suggestion
  width: 100%;
```
for more information check out the links in my comment on your html file!